Common/Sources/Helper/ThemeManager.swift

```diff
@@ -46,7 +46,7 @@ public class ThemeManager: NSObject {
             proxy.largeTitleTextAttributes =
                 [
                     NSAttributedString.Key.foregroundColor: Theme.NavigationBar.largeTitleColor,
-                    NSAttributedString.Key.font: Theme.NavigationBar.largeTitleFont
+                    NSAttributedString.Key.font: UIFont.boldSystemFont(ofSize: 17.0)
```

Comment:
If you are going to hardcode this value here, then what will happen if Integrators wants to change the font size and set `Theme.NavigationBar.largeTitleFont = UIFont.boldSystemFont(ofSize: 21.0)` but since we are not using Theme.NavigationBar.largeTitleFont here so how do you think that will work?